index.d.ts

```diff
@@ -21,6 +21,7 @@ declare namespace loadtest {
 		secureProtocol?: string;
 		statusCallback?(error: Error, result: any, latency: LoadTestResult): void;
 		contentInspector?(result: any): void;
+		indexParamCallBack?(): string;
```

Comment:
I take it that this is a new parameter for the API. Would you mind documenting it lightly in the `README.md`? An example would be great.